certified-connectors/EXPOCAD/apiDefinition.swagger.json

```diff
@@ -0,0 +1,4316 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "v3",
+    "title": "EXPOCAD",
+    "description": "This service interfaces with live EXPOCAD\u00ae event data.  Please use PUT, POST, and DELETE operations with discretion. Keep authorization codes private.  If your code has been compromised, contact tech@expocad.com immediately.",
+    "contact": {
+      "name": "EXPOCAD Developer Support",
+      "email": "developer@expocad.com"
```

Comment:
@mikefactorial Please add `website` as well in `contact` section

Documentation - [Adding Metadata](https://docs.microsoft.com/connectors/custom-connectors/certification-submission#adding-metadata)

Reply:
Added "url" per the documentation.